src/murmur/murmur_pch.h

```diff
@@ -80,22 +80,13 @@ extern "C" {
 #include <errno.h>
 #include <signal.h>
 #include <poll.h>
-#define INVALID_SOCKET -1
-#define SOCKET_ERROR -1
```

Comment:
Shouldn’t this file include the new `Utils.h` then? If not, the commit should describe why this is not necessary nor desired.

Reply:
Done.